src/hosting/api.ts

```diff
@@ -275,3 +275,82 @@ export async function createRelease(site: string, channel: string, version: stri
   );
   return res.body;
 }
+
+async function getAuthDomains(project: string): Promise<string[]> {
```

Comment:
None of these functions belong to the Hosting API. Could we move them to their own file? We have a catch-all `src/gcp/` folder that contains other GCP related things, maybe that would be a good candidate.